src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
+    display: none;
```

Comment:
Why class is named `visible` but in fact, you hide the element? It's inconsistent. Rename the class name